tests/cpp/test_matmul_scheduler.cpp

```diff
@@ -3260,13 +3260,13 @@ class HopperMatmulSchedulerTest
     // TODO cta tile is a multiple of mma macro for hopper.
     // Default cta_tile configuration is 2-CTA.
     gemm_tile.cta_tile =
-        GemmTile(2 * getM(mma_macro), getN(mma_macro), getK(mma_macro));
+        GemmTile(2 * getM(mma_macro), getN(mma_macro), 2 * getK(mma_macro));
 
     // TODO warp tile is (macroM, macroN, macroK) for hopper.
     gemm_tile.warp_tile =
-        GemmTile(getM(mma_macro), getN(mma_macro), getK(mma_macro));
+        GemmTile(getM(mma_macro), getN(mma_macro), 2 * getK(mma_macro));
 
-    mparams.supported_vec_size = {8, 8, 4};
+    mparams.supported_vec_size = {8, 8, 8};
```

Comment:
Good catch, although I think it currently has no meaning until we start handling epilogue inputs with supported vec size.